### ISSUE TITLE
Default deschedulerPolicy in helm chart causes crashloop

### ISSUE THREAD

**Issue by duplabe** [kubernetes-sigs/descheduler#1411]
**What version of descheduler are you using?**

descheduler version: v0.30.0

**Does this issue reproduce with the latest release?**

Yes.

**Which descheduler CLI options are you using?**

I'm using the helm chart v0.30.0

**Please provide a copy of your descheduler policy config file**

It's the default comming with the chart.

**What k8s version are you using (`kubectl version`)?**

<details><summary><code>kubectl version</code> Output</summary><br><pre>
$ kubectl version
Client Version: v1.30.1
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.30.0+k3s1
</pre></details>

**What did you do?**

I use the helm chart with the default options to deploy descheduler into my k8s cluster.

**What did you expect to see?**

Descheduler running successfully.

**What did you see instead?**

Descheduler throws config errors.

```
E0521 16:16:02.890228       1 run.go:74] "command failed" err="failed decoding descheduler's policy config \"/policy-dir/policy.yaml\": strict decoding error: unknown field \"nodeAffinityType\""
``` 

I will open a PR with my changes on the default profile.

**Comment by duplabe** [comment]
PR with possible fix: https://github.com/kubernetes-sigs/descheduler/pull/1412

**Comment by a7i** [comment]
Fixed in #1412 - we'll work on getting a v0.30.1 out in the next week or so

**Comment by duplabe** [comment]
it's in v0.30.1, thanks!
